hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -56,6 +56,7 @@ function HackathonForm(props: IProps) {
     })
     createHackathon(hackathonTitle, startDateValue!, endDateValue!).then((r) =>
       setTimeout(() => {
+        console.log(r)
```

Comment:
do we need to merge console.logs(r)? Also can we rename to `r` to fully qualified name? :D